c7n/resources/cw.py

```diff
@@ -304,3 +306,79 @@ def process_resource_set(self, client, accounts, resources):
             if found:
                 results.append(r)
         return results
+
+
+@LogGroup.action_registry.register('tag')
+class LogGroupAddTag(Tag):
+    """Action to create tag(s) on a resource
+
+    :example:
+
+    .. code-block:: yaml
+
+            policies:
+              - name: log-group-tag
+                resource: log-group
+                filters:
+                  - "tag:DesiredTag": absent
+                actions:
+                  - type: tag
+                    key: DesiredTag
+                    value: DesiredTagValue
+    """
+    permissions = ('logs:TagLogGroup',)
+
+    def process_resource_set(self, resources, tags):
+        client = local_session(self.manager.session_factory).client('logs')
+
+        def _add_tags(r):
+            try:
+                tag_dict = {}
```

Comment:
could simplify this with a comprehension, `tag_dict = {t['key']:t['value'] for t in tags}`

Reply:
ah, that's what i was looking for.